src/control/controlobject.h

```diff
@@ -110,6 +110,9 @@ class ControlObject : public QObject {
     // Instantly sets the value of the ControlObject
     static void set(const ConfigKey& key, const double& value);
 
+    // Instantly sets the value of the ControlObject to 1 and then to 0
+    static void toggle(const ConfigKey& key);
```

Comment:
I don't think this is needed. And why is this in a test PR?

Reply:
It is here because I use it for the test.
And I think that it can be quite useful since many controls need to be set back to 0 before they can be invoked again.

Reply:
> And I think that it can be quite useful since many controls need to be set back to 0 before they can be invoked again.

Which ones? I think most of them work even if they are already set to 1.0. Also, I'd argue that they should reset themselves to 0.0 instead of relying on external code to reset them.
@daschuer @uklotzde What do you think?

Reply:
The `recording_toggle` controls, same for `hotcue_X_activate` and many more (buttons). Setting it to 1 means pressed, back to 0 means released.

I don't think we should change anything about the behavior, it is fine as it is. You can't press a button while it is already pressed, so setting it back to 0 is vital to reproduce what is actually happening.